client/src/components/buttons/DeleteButton.js

```diff
@@ -4,33 +4,28 @@ import PropTypes from 'prop-types';
 import styled from 'styled-components';
 
 const Delete = styled.div.attrs({
-    className: "delete-item-btn"
+  className: 'delete-item-btn',
 })`
   color: #ff0000;
   cursor: pointer;
 `;
 
 class DeleteButton extends Component {
-    confirmDeleteItem = event => {
-        event.preventDefault();
-
-        if (
-            window.confirm(
-                `Do you want to permanently delete this item? ${this.props.id}`,
-            )
-        ) {
-            this.props.onDelete(this.props.id);
-        }
+  confirmDeleteItem = event => {
+    event.preventDefault();
 
+    if (window.confirm(`Do you want to permanently delete this item? ${this.props.id}`)) {
+      this.props.onDelete(this.props.id);
     }
+  };
```

Comment:
I have no idea why I thought the other way was easier to read 😅 Clearly, I overthought that.

Reply:
Which other way?

Reply:
Sorry I should've been more clear; I meant the other way it was formatted, which was like this:
```js
    if (
        window.confirm(
            `Do you want to permanently delete this item? ${this.props.id}`
        )
    ) {
         this.props.onDelete(this.props.id);
    }
```

Reply:
I put it back